content/en/blog/_posts/2022-03-31-ready-for-dockershim-removal.md

```diff
@@ -0,0 +1,32 @@
+
+## First, does this even affect you?
+
+If you are rolling your own cluster, or otherwise unsure whether or not this removal affects you, stay on the safe side and [check to see if you have any dependencies on Docker Engine](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/). Please note that using Docker Desktop to build your application containers is not a Docker dependency for your cluster. Container images built by Docker are compliant with the [Open Container Initiative (OCI)](https://opencontainers.org/), a Linux Foundation governance structure that defines industry standards around container formats and runtimes. They will work just fine on any container runtime supported by Kubernetes.
+
+If you are using a managed Kubernetes service from a cloud provider, and you haven’t explicitly changed the container runtime, it’s possible that there’s nothing else for you to do. Amazon EKS, Azure AKS, and Google GKE all default to containerd now, though if you have any node customizations, you should make sure they do not need updating. To check the runtime of your nodes, follow [Find Out What Container Runtime is Used on a Node](​​/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/).
+
+Regardless of whether you are rolling your own cluster or using a managed Kubernetes service from a cloud provider, you may need to [migrate telemetry or security agents that rely on Docker Engine](/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents/). 
+
+## I have a Docker dependency. What now?
+
+If your Kubernetes cluster does depend on Docker Engine and you intend to upgrade to Kubernetes v1.24 (which you should do for security), you will need to either change your container runtime from Docker Engine to something else, or use [cri-dockerd](https://github.com/Mirantis/cri-dockerd). Since [containerd](https://containerd.io/) is a graduated CNCF project and the runtime within Docker itself, it’s a safe bet. Fortunately, the Kubernetes project has already documented the process of [changing a node’s container runtime](/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd/), using containerd as the example. Instructions are similar for switching to one of the other supported runtimes.
```

Comment:
`suggestion
If your Kubernetes cluster does depend on Docker Engine and you intend to upgrade to Kubernetes v1.24 (which you should eventually do for security and similar reasons), you will need to either change your container runtime from Docker Engine to something else, or use [cri-dockerd](https://github.com/Mirantis/cri-dockerd). Since [containerd](https://containerd.io/) is a graduated CNCF project and the runtime within Docker itself, it’s a safe bet as an alternative container runtime. Fortunately, the Kubernetes project has already documented the process of [changing a node’s container runtime](/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd/), using containerd as the example. Instructions are similar for switching to one of the other supported runtimes.
```